README.md

```diff
@@ -2,33 +2,76 @@
 
 1.
   * Create a Human class that takes in a name and age.
-  * Add the function `ageOneYear` that ages the human. 
+  * Add the function `ageOneYear` that ages the human.
   * Add the function `eating`, that logs "mmm, mmm, mmm, I'm love'n it".
   * Create an instance of the Human class.
     * console log your humans age
     * call ageOneYear on your human
     * console log their age again.
     * call eating on your human.
-    
-2. 
+
+    class Human {
+     constructor(name, age){
+       this.name = name;
+       this.age = age;
+     }
+
+     ageOneYear(str){
+       return this.age++
+     }
+
+      eating(str){
+        return "mmm, mmm, mmm, I'm love'n it."
+      }
+    }
+
+
+    let daniel = new Human("Daniel", 34)
+    console.log(daniel.age)
+    daniel.ageOneYear()
+    console.log(daniel.age)
+    console.log(daniel.eating())
```

Comment:
This way, you only need to call on the function with `daniel.eating()` to log it into the console. Please see me if you have any questions about this or if I'm not explaining it clearly! :)